src/epd4in2/constants.rs

```diff
@@ -1,6 +1,7 @@
 #[rustfmt::skip]
 pub(crate) const LUT_VCOM0: [u8; 44] = [
-0x00, 0x17, 0x00, 0x00, 0x00, 0x02,        
+// 0x00, 0x17, 0x00, 0x00, 0x00, 0x02,        
+0x40, 0x17, 0x00, 0x00, 0x00, 0x02,        
```

Comment:
Is there a reason for this line change? If I remember correctly the first byte is for the amount of repetitions or length of the following line and it was 0x00 elsewhere in the referenced stuff I believe

Reply:
I noticed it in the video referenced above. No idea what the truth data is! Thoughts?

Reply:
After looking through multiple sources and Krasnows video once more I think it should be 0x00 there since all the other LUTs for VCOM0 also used only that voltage source and since it doesn't make a difference (or does it?) I would leave it the old way and maybe just add a note there if someone is ever interested in testing this extensively.